app/src/main/java/com/orange/ouds/app/ui/components/button/ButtonDemoScreen.kt

```diff
@@ -172,6 +194,31 @@ private fun ButtonDemoBox(colored: Boolean, modifier: Modifier = Modifier, conte
     }
 }
 
+@Composable
+private fun ButtonDemoCodeSnippet(state: ButtonDemoState, modifier: Modifier = Modifier) {
+    CodeSnippet(modifier = modifier) {
+        with(state) {
+            coloredBoxCall(onColoredBox) {
+                functionCall(OudsButton::class.simpleName.orEmpty()) {
+                    if (layout in listOf(ButtonDemoState.Layout.IconOnly, ButtonDemoState.Layout.IconAndText)) {
+                        constructorCallArgument<OudsButton.Icon>("icon") {
+                            painterArgument(R.drawable.ic_heart)
+                            contentDescriptionArgument(R.string.app_components_button_icon_a11y)
```

Comment:
Should we really display all the `stringResource(id=R.string.***)` for `contentDescription`? I find that it "pollutes" the code because it is very verbose. Maybe we can only display the string retrieved using the stringResource method? What do you think?